src/webextension/server.cpp

```diff
@@ -2,23 +2,25 @@
  * License, v. 2.0. If a copy of the MPL was not distributed with this
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
-#include "serverhandler.h"
+#include "./server.h"
 
 #include <QHostAddress>
 #include <QTcpSocket>
 
+#include "./connection.h"
```

Comment:
Can these be changed to `#include "connection.h"` and `#include "server.h"`? i.e. remove `./`